test/model/account/Account.spec.ts

```diff
@@ -57,6 +58,14 @@ describe('Account', () => {
         expect(account.address).to.not.be.equal(undefined);
     });
 
```

Comment:
Could you put in the unit test also some concrete test vectors?

Reply:
I've added some unit tests for the account